src/ManualInput/Input.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { forwardRef, useImperativeHandle, useRef, useState } from 'react';
+import { StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import { ManualInputDefault } from '../types';
+const obfusChars = ['/', ':'];
+type Props = {
+    defaultValue: ManualInputDefault;
+    onInputChange: (text: string) => void;
+    maxLength?: number;
+};
+export type FocusHandle = {
+    focus: () => void;
+};
+const Input = forwardRef<FocusHandle, Props>(({ defaultValue, onInputChange, maxLength }, ref) => {
+    const [value, setValue] = useState('');
+    const textInputRef = useRef<TextInput>(null);
+
+    useImperativeHandle(ref, () => ({
+        focus: () => {
+            textInputRef?.current?.focus();
+        },
+    }));
```

Comment:
Idly the `ref` passed in `forwardRef` should had been plugged into `TextInput` component, so user can call `textInputRef?.current?.focus();` from outside